backend/spring-routie/src/test/java/routie/routiespace/controller/RoutieSpaceControllerTest.java

```diff
@@ -38,4 +56,114 @@ public void createRoutieSpace() {
+                    "name": "스타벅스 강남점",
+                    "address": "서울시 강남구 테헤란로 123",
+                    "stayDurationMinutes": 60,
+                    "openAt": "09:00",
+                    "closeAt": "22:00",
+                    "breakStartAt": "15:00",
+                    "breakEndAt": "16:00",
+                    "closedDays": ["SUNDAY"]
+                }
+                """;
+
+        // when
+        Response response = RestAssured
+                .given()
+                .contentType(ContentType.JSON)
+                .body(requestBody)
+                .when()
+                .post("/routie-spaces/{routieSpaceIdentifier}/places", routieSpaceIdentifier)
+                .then()
+                .log().all()
+                .extract().response();
+
+        HttpStatus actualHttpStatus = HttpStatus.valueOf(response.getStatusCode());
+        HttpStatus expectedHttpStatus = HttpStatus.OK;
+
+        Long placeId = response.jsonPath().getLong("id");
+
+        // then
+        assertThat(expectedHttpStatus).isEqualTo(actualHttpStatus);
+        assertThat(placeId).isNotNull();
+        assertThat(placeId).isPositive();
+    }
+
+    @Test
+    @DisplayName("장소 목록 조회에 성공한다")
+    public void readPlaces() {
+        // given
+        String place1RequestBody = """
+                {
+                    "name": "스타벅스 강남점",
+                    "address": "서울시 강남구 테헤란로 123",
+                    "stayDurationMinutes": 60,
+                    "openAt": "09:00",
+                    "closeAt": "22:00",
+                    "breakStartAt": "15:00",
+                    "breakEndAt": "16:00",
+                    "closedDays": ["SUNDAY"]
+                }
+                """;
+
+        String place2RequestBody = """
+                {
+                    "name": "투썸플레이스 역삼점",
+                    "address": "서울시 강남구 역삼동 789",
+                    "stayDurationMinutes": 90,
+                    "openAt": "08:00",
+                    "closeAt": "23:00",
+                    "breakStartAt": null,
+                    "breakEndAt": null,
+                    "closedDays": ["MONDAY", "TUESDAY"]
+                }
+                """;
+
+        RestAssured
+                .given()
+                .contentType(ContentType.JSON)
+                .body(place1RequestBody)
+                .when()
+                .post("/routie-spaces/{routieSpaceIdentifier}/places", routieSpaceIdentifier);
+        RestAssured
+                .given()
+                .contentType(ContentType.JSON)
+                .body(place2RequestBody)
+                .when()
+                .post("/routie-spaces/{routieSpaceIdentifier}/places", routieSpaceIdentifier);
+
+        // when
+        Response response = RestAssured
+                .when()
+                .get("/routie-spaces/{routieSpaceIdentifier}/places", routieSpaceIdentifier)
+                .then()
+                .log().all()
+                .extract().response();
```

Comment:
응답도 요청과 마찬가지로 Response 클래스를 그대로 사용해서 받아올 수 있어요!

```suggestion
        PlaceListResponse response = RestAssured
                .when()
                .get("/routie-spaces/{routieSpaceIdentifier}/places", routieSpaceIdentifier)
                .then()
                .log().all()
                .extract()
                .as(PlaceListResponse.class);
```

Reply:
위의 코멘트와 더불어서 같이 남기겠습니다.

픽스처 유지보수, 타입 안정성을 위해서 클래스를 사용하는 것도 좋지만, 
E2E 테스트에서 실제 클라이언트의 request body를 모방하는 것이 더 정확하지 않을까 생각해보았어요.
실제 요청에 대한 DTO 유효성 검사까지 할 수 있으니까요!

헤일러의 방법도 한번 고려해보고 장단점을 분석해볼게요 👍